### PR TITLE
Add explore api calls

### PR DESCRIPTION
**Pull request by stevensonmat2**
- add two api calls to explore page
- add data parsing/formatting
- add html to render

### CODE REVIEW

File: gun-violence-dashboard/src/pages/Compare.js [hunk 1]
```diff
@@ -1,5 +1,8 @@
 import React from "react";
 
+// use world ranking where possible
```
Comment (nicolekurtz93) [inline gun-violence-dashboard/src/pages/Compare.js:3]: Let's remove these comments

File: gun-violence-dashboard/package.json [hunk 2]
```diff
@@ -46,4 +46,4 @@
       "last 1 safari version"
     ]
   }
```
Comment (nicolekurtz93) [inline gun-violence-dashboard/package.json]: I'm not able to run this code without npm installing a bunch of libraries myself. I'm getting a bunch of missing package issues. Maybe try removing node_modules and npm install to see if it works on your machine from scratch.

File: gun-violence-dashboard/src/pages/Explore.js [hunk 3]
```diff
@@ -1,9 +1,88 @@
-import React from "react";
```
Comment (nicolekurtz93) [inline gun-violence-dashboard/src/pages/Explore.js:1]: On my end, you need this import for your code to run.

File: gun-violence-dashboard/src/pages/Explore.js [hunk 4]
```diff
@@ -1,9 +1,88 @@
...
+  const handleLocationChange = (selectedOption) => {
+    setSelectedLocation(selectedOption);
+  };
+
+  return (
+    <div className='d-flex flex-column justify-content-center align-items-center m-2'>
+      <h1>Explore By Location</h1>
+      <h2>{country}</h2>
+
+      <Select
```
Comment (nicolekurtz93) [inline gun-violence-dashboard/src/pages/Explore.js:99]: You will need to add a corresponding label for this dropdown.

File: gun-violence-dashboard/src/pages/Explore.js [hunk 5]
```diff
@@ -1,9 +1,88 @@
...
+        options={locations}
+        value={selectedLocation}
+        onChange={handleLocationChange}
+        placeholder="Select a location"
+        isSearchable
+      />
+      <div className='d-flex justify-content-between'>
+        <div>
+          <h3>Gun Ownership</h3> 
+          {gunsOwned}
```
Comment (nicolekurtz93) [inline gun-violence-dashboard/src/pages/Explore.js]: You mention in your description that this includes formatting. If that's correct, I'm seeing some weird formatting behavior. See screenshot: 
matt1

File: gun-violence-dashboard/src/pages/Explore.js [hunk 6]
```diff
@@ -1,9 +1,88 @@
...
+  }, []);
+
+  useEffect(() => {
+    if (selectedLocation) {
+      const fetchData = async () => {
+        try {
+          const response = await axios.get(gunsOwnedUrl);
+          setGunsOwned(formatGunOwnershipData(response.data.result.columnValue));
+          setCountry(response.data.result.location);
+          console.log(response.data);
```
Comment (nicolekurtz93) [inline gun-violence-dashboard/src/pages/Explore.js]: Remove console

**Review by nicolekurtz93 — CHANGES_REQUESTED** [review]
Good job, Matt. I had trouble running your code, so maybe try deleting all node_modules and 'npm install' to see if it works on your end. See my additional comments.

File: gun-violence-dashboard/src/pages/Explore.js (hunk 5 above)
Comment (stevensonmat2) [reply gun-violence-dashboard/src/pages/Explore.js]: I'm just formatting/parsing the data from the api at this point

File: gun-violence-dashboard/src/pages/Explore.js (hunk 4 above)
Comment (stevensonmat2) [reply gun-violence-dashboard/src/pages/Explore.js:67]: I'll do an acess check later on

File: gun-violence-dashboard/src/pages/Compare.js (hunk 1 above)
Comment (stevensonmat2) [reply gun-violence-dashboard/src/pages/Compare.js:3]: I'm still working so I want these notes, they can be removed later

File: gun-violence-dashboard/src/pages/Explore.js (hunk 3 above)
Comment (stevensonmat2) [reply gun-violence-dashboard/src/pages/Explore.js:1]: works fine on my end

File: gun-violence-dashboard/src/pages/Explore.js (hunk 6 above)
Comment (stevensonmat2) [reply gun-violence-dashboard/src/pages/Explore.js:48]: this is still in progress, will remove later

File: gun-violence-dashboard/src/pages/Explore.js (hunk 4 above)
Comment (nicolekurtz93) [reply gun-violence-dashboard/src/pages/Explore.js:99]: We really shouldn't be committing code to master with these kinds of issues.

**Review by nicolekurtz93 — APPROVED** [review]
Looks good! Thanks for fixing those issues.

**Pull request merged**